### PR TITLE
Carto25 20191216

### PR DESCRIPTION
**Pull request by manmorjim**
Merged community `stable-2.5` as of 2019-12-16

### CODE REVIEW

File: raster/rt_pg/rtpg_mapalgebra.c [hunk 1]
```diff
@@ -78,10 +78,14 @@ Datum RASTER_mapAlgebra2(PG_FUNCTION_ARGS);
...
 
 /* Quiet warning */
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunknown-pragmas"
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wgnu-variable-sized-type-not-at-end"
+#if defined(__GNUC__)
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wunknown-pragmas"
+#endif
```
Comment (manmorjim) [inline raster/rt_pg/rtpg_mapalgebra.c]: @Algunenano The GCC directives were added in the CARTO's fork so I added the `__GNUC__` checking for them.

Comment (Algunenano) [reply raster/rt_pg/rtpg_mapalgebra.c]: In that case we should leave it as it's defined currently in master:
https://github.com/postgis/postgis/blob/master/raster/rt_pg/rtpg_mapalgebra.c#L80-L103

Comment (manmorjim) [reply raster/rt_pg/rtpg_mapalgebra.c]: :+1:  done

File: CHANGELOG.carto.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 2.5.3.2+carto-1
+
+**Release date**: 2019-12-16
+
+Changes:
+- Backport [a4a040c](https://github.com/postgis/postgis/commit/a4a040c4949089f32b63e6a647d1d9696992a132): Avoid throwing during update if st_union(geometry) doesn't exist.
```
Comment (Algunenano) [inline CHANGELOG.carto.md]: No need to mention individual backports if they are included in the merge of `stable-2.5`.

**Review by Algunenano — APPROVED** [review]
LGTM. Please remember to update Version.config file too.

**Pull request merged**